dart/lib/src/sentry_options.dart

```diff
@@ -278,6 +278,8 @@ class SentryOptions {
   ///   because the connection was interrupted.
   /// Use with [SentryHttpClient] or `sentry_dio` integration for this to work,
   /// or iOS native where it sets the value to `enableCaptureFailedRequests`.
+  @Deprecated(
+      "All Request and Responses are now logged, if `sendDefaultPii` is `true` and `maxRequestBodySize` and `maxResponseBodySize` conditions are met.")
   bool captureFailedRequests = true;
```

Comment:
let's rethink this, I don't think deprecating this flag is the right way to go

Reply:
we still want to provide this flag to disable capturing of failed network requests completely, so yes we shouldn't deprecate this